package.json

```diff
@@ -9,7 +9,8 @@
     "build": "ng build --configuration production",
     "build-aot-disable": "ng build --configuration production --aot=false --build-optimizer=false",
     "docker-build": "/bin/bash docker-build/build.sh",
-    "postinstall": "ngcc"
+    "postinstall": "ngcc",
+    "add-repo-tag": "make add-repo-tag"
```

Comment:
Do we need that in packages file? Do we have a scenario when we want to run it via JS, not via bash command `make add-repo-tag` directly?

Reply:
I thought it was a good idea to have one entry point to control the project

Reply:
There is no special technical reason to add a launch via npm ng

Reply:
Maybe in that case in README should be only npm variant, so if we need to change realisaztion (from example moving from Makefile to something else) in all cases we will use NPM command